sdk/src/system_transaction.rs

```diff
@@ -17,30 +34,78 @@ pub fn create_account(
     space: u64,
     program_id: &Pubkey,
 ) -> Transaction {
+    Transaction::new_signed_instructions(
+        &[from_keypair],
+        create_account_instructions(from_keypair, to, lamports, space, program_id),
+        recent_blockhash,
+    )
+}
+
+pub fn create_account_with_dummy_sig(
+    from_keypair: &Keypair,
+    to: &Pubkey,
+    recent_blockhash: Hash,
+    lamports: u64,
+    space: u64,
+    program_id: &Pubkey,
+) -> Transaction {
+    Transaction::new_signed_instructions_random(
```

Comment:
API naming: At this layer, we translate to more higher and use-oriented abstract concept from _random_ (implementation details) to _dummy_ (users just want non-verifable fake signatures cheaply).